pylint/checkers/classes/class_checker.py

```diff
@@ -1454,6 +1459,24 @@ def _check_invalid_overridden_method(
                 node=function_node,
             )
 
+    def _check_functools_or_not(self, decorator: nodes.Attribute) -> bool:
+        if decorator.attrname != "cached_property":
+            return False
+
+        if not isinstance(decorator.expr, nodes.Name):
+            return False
```

Comment:
This line is easily coverable by testing with a decorator that's a Call, like `@cached_property()`

Reply:
@kyoto7250 I think this line is still missing coverage. You can add a test case with a decorator that's a call. Probably better not to use `cached_property()` as the fake call, since that could be confusing.

Reply:
I tried https://github.com/pylint-dev/pylint/pull/8758/commits/a3827965c6e1461f5c0859a4da55cafb6f26666e, but This doesn't seem to be a valid in python3.8.

The test here is a little difficult. `L1261` confirms whether it is an `nodes.Attribute`, so I think that this conditional branch will not be entered in if we only use `nodes.Call`.

Reply:
That's actually the correct test, and if you want to verify the line is executed, just insert a bullet like `1 / 0` before the return and watch it fail.

> This doesn't seem to be a valid in python3.8.

That's okay! Seems to be a legitimate syntax difference only enabled on 3.9+ and up. The test case is still a good one, you'll just need to move it to a new file suffixed with _py39.py and with a corresponding file ending in `.rc` with the min-version set to py 3.9. `method-cache-max-size-none` has tests set up exactly so.

Reply:
Thank you for your comment.
I commited https://github.com/pylint-dev/pylint/pull/8758/commits/70615c29f520a45ecd70ae2b6ffa080d73bb5792